### DISCUSSION TITLE
Proposal for a new feature

### DISCUSSION THREAD

**Discussion by harshgarg22** [google/or-tools#4476] [Feature requests]
hey @lperron and @Mizux

i am harsh garg, a CS junior based in dubai. i interned at a major airline company here, this summer and made extensive use of google or tools, particularly the CP-SAT solver to optimize seat allocation over different legs. i've never made any open-source contributions and when i went through the contribution guidelines, you asked to reach out in case of any project proposals. i think the library could really benefit from functions to solve the _**transportation problem (both balanced and unbalanced)**_. let me know what your views are, and then i can start contributing to the repo.

best,
harsh :)

**Comment by Mizux** [comment]
seems related to https://cloud.google.com/customers/swiss

**Comment by harshgarg22** [comment]
are there any new features that you are working on? i would love to contribute to them. @Mizux 

best,
harsh
